lib/confirmation.rb

```diff
@@ -0,0 +1,22 @@
+require 'twilio-ruby'
+
+class Confirmation
+
+  def send_text
```

Comment:
Could this method be simplified and refactored into smaller methods each with a single purpose?